api/api.go

```diff
@@ -116,7 +116,7 @@ type Battery interface {
 
 // BatteryCapacity provides a capacity in kWh
 type BatteryCapacity interface {
-	Capacity() float64
+	Capacity() (float64, error)
```

Comment:
I aligned this with all the other getter functions that can fail. The downside of the current implementation is that whenever `Capacity()` is accessed a new attempt to retrieve the value is made. Since this is something that should never change the more logical approach would be to only load this once at startup and then store it.
Should we keep `Capacity() float64` and abort with the error in `meter.NewConfigurableFromConfig`?

Reply:
> Should we keep Capacity() float64 and abort with the error in meter.NewConfigurableFromConfig?

It needs the error like anything else read at runtime.

Reply:
Ich bin unsicher. Capacity ist ja tatsächlich statisch. Also was spräche dagegen, es beim Anlegen auszulesen? Ebenso wie z.B. den Nickname des Fahrzeugs? Das würde diesen PR völlig verändern ;)